### PR TITLE
improve user search page #1660

### LINKED ISSUE #1660: Improve User-search page
**Describe the bug**
Make the User-search page in sync with Project-search page. This requires the following changes:
--Add border around individual search result
--Remove the horizontal rule between results
--Center and add padding to the pagination element (to be done on both Project and User search page)
image
--Fix names moving on to the side of profile image
image

**Are you working on this issue? (Yes/No)**
Maybe

### PR DESCRIPTION
**Pull request by pavanjoshi914**
Fixes #
closes #1660 

#### Describe the changes you have made in this PR -
   1>user search page in sync with project search page
   2>fix applied, so that user name always remain at appropriate position (below image)
   3>if user name is too long it will not go outside container
   4>aligning pagination to center
   

### Screenshots of the changes (If any) -
Screenshot from 2020-09-20 19-59-46

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by pavanjoshi914** [comment]
@aayush-05 have a review on this ,if any improvements ? let me know

File: app/assets/stylesheets/search.scss [hunk 1]
```diff
@@ -137,6 +137,13 @@
   font-weight: 500;
 }
 
+.username-contain {
```
Comment (aayush-05) [inline app/assets/stylesheets/search.scss:140]: Prefix classname with search. Maybe add text-overflow: ellipsis as well to signify shortened display name

File: app/views/users/logix/search.html.erb [hunk 2]
```diff
@@ -12,11 +12,13 @@
 <% else %>
   <div class="container search-container">
     <% @results.map { |user| ProfileDecorator.new(user) }.each do |profile| %>
-      <div class="row center-row search-user-container">
+      <div class="row center-row search-user-container border mb-4">
```
Comment (aayush-05) [inline app/views/users/logix/search.html.erb:15]: Purpose of "border"? Also can we move mb-4 to search-user-container css if its not conflicting with other styles

**Review by aayush-05 — CHANGES_REQUESTED** [review]

File: app/views/users/logix/search.html.erb (hunk 2 above)
Comment (aayush-05) [reply app/views/users/logix/search.html.erb:15]: Also, I think you will be required to replace search-user-container with search-card-container to add a border to individual card. See if we are using search-user-container anywhere else or remove it

File: app/assets/stylesheets/search.scss (hunk 1 above)
Comment (pavanjoshi914) [reply app/assets/stylesheets/search.scss:140]: made changes accordingly, resolved!!

File: app/views/users/logix/search.html.erb (hunk 2 above)
Comment (pavanjoshi914) [reply app/views/users/logix/search.html.erb:15]: search-user-container is not used at anywhere else so removing class from erb as well as stylesheet
I thought of using search-card-container but it is making each card as big as cards in project search
smaller usercards looked more tiny to me!!
 so i settled up with adding border manually
changed search-user-container with search-card-container

Comment (aayush-05) [reply app/views/users/logix/search.html.erb]: Define something like search-user-card-container and use that. We require cards with less width in this page as earlier. That class would account for everything from width to border

Comment (pavanjoshi914) [reply app/views/users/logix/search.html.erb]: Screenshot from 2020-10-22 12-37-43

**Comment by pavanjoshi914** [comment]
@aayush-05  please have a review on this.. thanks!!

File: app/views/users/logix/search.html.erb (hunk 2 above)
Comment (aayush-05) [reply app/views/users/logix/search.html.erb]: Everything else looks great; @pavanjoshi914 can we just decrease the size of the user image a bit and move all content in each card a bit below, adding more padding to the top.

Comment (pavanjoshi914) [reply app/views/users/logix/search.html.erb]: Ohk making following changes thanks!!

**Comment by pavanjoshi914** [comment]
new pr for this with some extra changes #2030

**Pull request closed without merging**